Tests.Integration.Presentation.Web/Interfaces/ItIterfacesTest.cs

```diff
@@ -100,6 +100,28 @@ public async Task Cannot_Add_Data_Row(OrganizationRole role)
             }
         }
 
+
+
+        [Theory]
+        [InlineData(OrganizationRole.GlobalAdmin)]
+        public async Task Author_Is_Same_As_Exhibit_System(OrganizationRole role)
+        {
+            //Arrange
+            var login = await HttpApi.GetCookieAsync(role);
+            const int organizationId = TestEnvironment.DefaultOrganizationId;
+            var interfaceDto = await InterfaceHelper.CreateInterface(InterfaceHelper.CreateInterfaceDto(A<string>(), A<string>(), null, organizationId, AccessModifier.Public));
+            var system = await ItSystemHelper.CreateItSystemInOrganizationAsync(A<string>(), organizationId, AccessModifier.Public);
+
+            //Act - perform the action with the actual role
+            await InterfaceExhibitHelper.CreateExhibit(system.Id, interfaceDto.Id, login);
+            var interfaceResult = await InterfaceHelper.GetInterfaceById(interfaceDto.Id, login);
```

Comment:
de to "getter" kan du flytte ned i "assert" blokken. De er jo ikke en del af "act" delen